### PR TITLE
mapPropsToRequestsToProps is called with context as 2nd argument (closes #69)

### LINKED ISSUE #69: Add context to the parameters of the mapping function
Why don't we also pass context as a second parameter to the data-mapping function?
I looks like only props are passed:
https://github.com/heroku/react-refetch/blob/master/src/components/connect.js#L156

I wish I could also access the context like so:
```javascript
DataSourceEditModal = connect((props, context) => {

  // .... check context for router or other state in it. ....

  return { ...mapping... };
}
```

### PR DESCRIPTION
**Pull request by nfcampos**
@ryanbrainard any other test you can think is appropriate?
I'm testing
- context is indeed provided as the 2nd argument
- when context changes mapPropsToRequestsToProps is called with the new context

### CODE REVIEW

File: test/components/connect.spec.js [hunk 1]
```diff
@@ -587,6 +587,69 @@ describe('React', () => {
...
     })
 
+    it('should invoke mapPropsToRequestsToProps with context', () => {
+      let contextPassedIn = []
+
+      @connect((props, context) => {
+        contextPassedIn.push(context)
+        return {}
+      })
+      class WithProps extends Component {
```
Comment (ryanbrainard) [inline test/components/connect.spec.js]: maybe rename this to `InnerComponent` since it actually doesn't have any props

**Comment by ryanbrainard** [comment]
Looks good! Can you update the API doc and maybe an example in the readme?

File: test/components/connect.spec.js (hunk 1 above)
Comment (nfcampos) [inline test/components/connect.spec.js]: good point, I've renamed it.

**Comment by nfcampos** [comment]
I'll update the docs.
As for the example, any example with context gets kind of verbose (defining all those contextTypes, etc.), which might be too much for the readme, no?

**Comment by ryanbrainard** [comment]
> As for the example, any example with context gets kind of verbose (defining all those contextTypes, etc.), which might be too much for the readme, no?

Yeah, perhaps. I'll merge this now and then we can add more to the readme if it adds value.

Thanks for working on this @nfcampos !

**Comment by nfcampos** [comment]
happy to help!

**Comment by ryanbrainard** [comment]
Released in https://github.com/heroku/react-refetch/releases/tag/v0.8.0-beta.0

**Pull request merged**